aws_quickstart/main_v2.yaml

```diff
@@ -0,0 +1,152 @@
+      If this template was launched from the Datadog app, this key is tied to the user that launched the template,
+      and is a key specifically generated for this integration.
+    Type: String
+    Default: ""
+  DatadogSite:
+    Type: String
+    Default: datadoghq.com
+    Description: Define your Datadog Site to send data to.
+    AllowedValues:
+      - datadoghq.com
+      - datadoghq.eu
+      - us3.datadoghq.com
+      - us5.datadoghq.com
+      - ddog-gov.com
+  IAMRoleName:
+    Description: Customize the name of IAM role for Datadog AWS integration
+    Type: String
+    Default: DatadogIntegrationRole
+  InstallLambdaLogForwarder:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+    Description: >-
+      Determines whether the default configuration for the Datadog Lambda Log Forwarder is installed 
+      as part of this stack. This is useful for sending logs to Datadog for use in Log Management or Cloud SIEM. 
+      Customers who want to customize this setup to include specific custom tags, data scrubbing or redaction rules,
+      or send logs using AWS PrivateLink should select “no” and install this independently
+      (https://docs.datadoghq.com/serverless/libraries_integrations/forwarder/#installation).
+    Default: true
+  DisableMetricCollection:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+    Description: >-
+      Disabling metric collection for this account will lead to a loss in visibility into your AWS services. 
+      Disable this if you only want to collect tags or resource configuration information from this AWS account, 
+      and do not want to use Datadog Infrastructure Monitoring.
+    Default: false
+  CloudSecurityPostureManagement:
+    Type: String
+    AllowedValues:
+      - true
+      - false
+    Description: >-
+      Add the AWS Managed SecurityAudit policy to your Datadog AWS Integration role, and enable 
+      Datadog Cloud Security Posture Management (CSPM) to start performing configuration checks across your AWS account.
+      Datadog CSPM is a product that automatically detects resource misconfigurations in your AWS account according to 
+      industry benchmarks. More info: https://www.datadoghq.com/product/security-platform/cloud-security-posture-management/
+    Default: false
+Conditions:
+  InstallForwarder:
+    Fn::Equals:
+      - !Ref InstallLambdaLogForwarder
+      - true
+  # A workaround for being unable to check list contains
+  CloudSecurityPostureManagementPermissions:
+    Fn::Equals:
+      - !Ref CloudSecurityPostureManagement
+      - true
+Resources:
+  # A Macro used to generate policies for the integration IAM role based on user inputs
+  DatadogAPICall:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      TemplateURL: "https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/datadog_integration_api_call_v2.yaml"
```

Comment:
I see that we have the bucket hard-coded here, but we are using `BUCKET_PLACEHOLDER` below.  Looks like this is how it was before, but I think you should be able to use the placeholder here and the release script subs it in.

Reply:
Yeah this'll work with BUCKET_PLACEHOLDER, thanks for the catch I'll change that.